src/taco-cli/cli/tacoErrorCodes.ts

```diff
@@ -4,9 +4,6 @@
 // Error Codes: 5000- 5999
 enum TacoErrorCode {
     // General errors 5000-5449
-    CordovaCmdNotFound = 5001,
```

Comment:
I would leave these errors here, with comments indicating that they have been moved (or maybe change them to _CordovaCmdNotFound so we don't accidentally autocomplete to them)

Otherwise it'll be very hard to not reuse them, especially things like 5015 and 5016 which were at the end of their block